cypress/integration/tasks.ts

```diff
@@ -12,42 +12,45 @@ describe("App", function () {
   })
   it("shows a modal after clicking Create New Task button and hides it after clicking cancel", function () {
     cy.visit("/todo")
-    cy.get(".createTaskButton").click()
+    cy.get("[data-testid=createButton]").click()
```

Comment:
If you have https://testing-library.com/docs/cypress-testing-library on this project you could use `getBytestId` here as a cleaner way of doing this. Another option is to use `findByText` to look for the contents of the button from the perspective of a user.

Reply:
🎉